Integrations/integration-Alexa.yml

```diff
@@ -0,0 +1,145 @@
+description: Alexa provides website ranking information that can be useful in determining
+  if the domain in question has a strong web presence.
+detaileddescription: To use this integration, simply call ```!domain```. !domain
+  accepts "domain" as a parameter to query. Threshold is the point where a domain will be considered unknown versus suspicious.
+configuration:
+- display: Sensitivity threshold for configuring which domains are suspicious versus
+    trusted.
+  name: threshold
+  defaultvalue: "2000000"
+  type: 0
+  required: true
+script:
+  script: |-
+    import urllib, sys, re
+    import xml.etree.ElementTree as ET
+    from collections import defaultdict
+
+
+    """GLOBAL VARIABLES/CONSTANTS"""
+    THRESHOLD = int(demisto.params().get('threshold'))
+
+
+    """HELPER FUNCTIONS"""
+    def get_xml(domain):
+        xml = urllib.urlopen('http://data.alexa.com/data?cli=10&dat=s&url={}'.format(domain)).read()
+        return xml
+
+
+    """MAIN FUNCTIONS"""
+    def alexa_domain(domain):
+        xml = get_xml(domain)
+        root = ET.fromstring(xml)
+        return root
+
+
+    """COMMAND FUNCTIONS"""
+    def alexa_domain_command():
+        domain = demisto.args().get('domain')
+        dbot_score_text = 'unknown'
+        xml = get_xml(domain)
```

Comment:
lets update the command function to that it will return the xml.
I'm just concerned that we may call the API twice and get different result on each call

Reply:
I originally had it doing that, but for some reason when I return it as part of the ```alexa_domain``` function (using a tuple return) it crashes circleci.